### ISSUE TITLE
If I have three class to seg,how to set?Like this?

### ISSUE THREAD

**Issue by HuangQinJian** [qubvel/segmentation_models.pytorch#20]

**Comment by LuZhenHuan** [comment]
how to set more than one classes？

**Comment by qubvel** [comment]
@LuZhenHuan read at least README before open an issue

**Comment by LuZhenHuan** [comment]
sorry about that, train procees is OK when set more than one classes. But i dont know how to visualize two classes or more.

**Comment by qubvel** [comment]
Visualize them one by one or take an argmax of classes axis

**Comment by LuZhenHuan** [comment]
got it, thanks

**Comment by hust-kevin** [comment]
@LuZhenHuan how to modify code to train more than one class? the loss function  of this repo is for binary classification
